mysql-schema/028-group-member.sql

```diff
@@ -0,0 +1,11 @@
+SET NAMES utf8mb4;
+START TRANSACTION;
+
+INSERT INTO changelog VALUES (28, UNIX_TIMESTAMP());
+
+CREATE TABLE group_members (
+  user_id INT UNSIGNED NOT NULL,
```

Comment:
Any MySQL table that you create should always have a primary key. In this case, a combination of user_id/group_id will only appear once so you should make a primary key with those 2 fields.